docker/precompiled/Dockerfile

```diff
@@ -1,29 +1,43 @@
-FROM julia:latest
+FROM julia:latest as builder
+
+RUN apt-get update \
+    && apt-get -y install clang \
+    && rm -rf /var/lib/apt/lists/*
 
 ENV USER pluto
 ENV USER_HOME_DIR /home/${USER}
+ENV DEBIAN_FRONTEND=noninteractive
 ENV JULIA_DEPOT_PATH ${USER_HOME_DIR}/.julia
 ENV NOTEBOOK_DIR ${USER_HOME_DIR}/notebooks
-ENV JULIA_NUM_THREADS = 100
+ENV JULIA_NUM_THREADS=100
```

Comment:
I have just seen that using "=" for variable assignment in Dockerfile does not work.
This line should be changed to
`ENV JULIA_NUM_THREADS 100`
Same for here:
`ENV DEBIAN_FRONTEND noninteractive`
For the non-precompiled Docker image I just corrected it.

Reply:
thanks! no wonder it wasn't picking up 😅